contribs/gmf/src/directives/search.js

```diff
@@ -135,7 +136,8 @@ gmf.searchDirective = function(gmfSearchTemplateUrl) {
       'clearbutton': '=gmfSearchClearbutton',
       'colorchooser': '=gmfSearchColorchooser',
       'coordinatesProjections': '=?gmfSearchCoordinatesprojections',
-      'additionalListeners': '=gmfSearchListeners'
+      'additionalListeners': '=gmfSearchListeners',
+      'maxZoom': '<gmfMaxZoom'
```

Comment:
With my previous comments:

`'maxZoom': <gmfSearchMaxzoom`

(/!\ watch out the uppercase)